### DISCUSSION TITLE
Telegram integration

### DISCUSSION THREAD

**Discussion by supperka** [arendst/Tasmota#12446] [Q&A] [answered]
Hello ,
All my telegram integration on my tasmota devices , stopped working.
I tried revoke token and put new in config ,but still I'm getting error.
"Send failed"
Tmstate is on,on,off
 all worked fine more than year.

**Comment by ascillato** [comment, marked as answer]
Please, open a new issue with all the required information in order to be able to help you. All the logs and everything asked in the template. Thanks.

**Reply by ajithvasudevan** [reply]
I have opened issue #12451

**Reply by supperka** [reply]
Thank you for issue open.
@ajithvasudevan
